internal/connectors/db/process_result_set.go

```diff
@@ -1,31 +1,69 @@
 package db
 
 import (
+	"errors"
 	"fmt"
-	"ydbcp/internal/types"
-
+	"github.com/google/uuid"
 	"github.com/ydb-platform/ydb-go-sdk/v3/table/result"
 	"github.com/ydb-platform/ydb-go-sdk/v3/table/result/named"
+	"ydbcp/internal/types"
 )
 
 type StructFromResultSet[T any] func(result result.Result) (*T, error)
 
 type InterfaceFromResultSet[T any] func(result result.Result) (T, error)
 
+func StringOrDefault(str *string, def string) string {
+	if str == nil {
+		return def
+	}
+	return *str
+}
+
+func StringOrEmpty(str *string) string {
+	return StringOrDefault(str, "")
+}
+
 func ReadBackupFromResultSet(res result.Result) (*types.Backup, error) {
 	var (
-		backupId    types.ObjectID
-		operationId types.ObjectID
+		backupId     [16]byte
+		containerId  string
+		databaseName string
+		s3endpoint   *string
+		s3region     *string
+		s3pathprefix *string
+		status       *string
 	)
 
 	err := res.ScanNamed(
 		named.Required("id", &backupId),
-		named.Optional("operation_id", &operationId),
+		named.Required("container_id", &containerId),
+		named.Required("database", &databaseName),
+		named.Optional("s3_endpoint", &s3endpoint),
+		named.Optional("s3_region", &s3region),
+		named.Optional("s3_path_prefix", &s3pathprefix),
+		named.Optional("status", &status),
 	)
 	if err != nil {
 		return nil, err
 	}
-	return &types.Backup{ID: backupId}, nil
+
+	id, err := uuid.FromBytes(backupId[:])
```

Comment:
There is a strange behaviour. I use nbstream ydb docker:
![image](https://github.com/user-attachments/assets/93895450-6306-432c-a6c7-153186fcd3c4)
see `id` and `status` field displays. `status` is broken, `id` is correct.
But when i query `id` via SDK, it reads following values:
```
qrort@qrort-Latitude-5511:~/ydbcp$ ./test.sh ListBackups
{
  "backups": [
    {
      "id": "c830d44f-c000-b480-11d1-9dad6ba7b810",
      "status": "PENDING"
    },
    {
      "id": "78563412-7856-3412-5678-123412345678"
    }
  ],
  "nextPageToken": "2"
}
```
The bytes get mixed somehow. I tried different convert methods, but to no avail.